### ISSUE TITLE
Add information to the download page to more explicitly state which versions have parallel support.

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#868]
Someone recently downloaded a pre-built linux version and then tried to use libsim in parallel with it and it failed because the pre-built version doesn't have parallel support. We should make that more explicit. The windows and Mac versions have parallel support. The windows version may require mpi to be installed, not 100% for sure.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 02/06/2013 10:57 am
Original update: 03/10/2014 08:14 pm
Ticket number: 1331
